test_suite/generic/test_generic.py

```diff
@@ -16,6 +16,11 @@ def test_no_avc_denials(self, host, instance_data):
         """
         Check there is no avc denials (selinux).
         """
+        if host.system_info.distribution == 'rhel' and \
+                float(host.system_info.release) == 9.4:
+            pytest.skip('RHEL-24346: Skipping on RHEL-9.4 due to a known issue with NetworkManager.')
```

Comment:
Q: is there a corresponding ticket to re-enable this check once selinux-policy is fixed ? Ideally we'd like to have this execute in future versions of 9.4 instead of being skipped for the entire duration of he minor release.

Reply:
+1. I created this ticket for future reference. We will see it each time we review backlog items (at least once per Q).
https://issues.redhat.com/browse/CLOUDX-848